### DISCUSSION TITLE
Help with prompt and settings

### DISCUSSION THREAD

**Discussion by krassheit** [lllyasviel/Fooocus#3636] [General]
Hi there,
I am still a beginner with Fooocus. I tried to render a picture like this but didn't succeed.
Anyone who could give me some ideas on the prompt and settings? Is juggernaut the right model?

Thank you!

![IMG_4302](https://github.com/user-attachments/assets/5a5be6fd-67ab-4af0-b4df-0989a90341a1)

**Comment by Tomcreating** [comment]
What styles have you tested? Looks kind of like cyberpunk. Have you tried using the in Describe function?
